indexer/displacement_manager.hpp

```diff
@@ -174,11 +174,11 @@ class DisplacementManager
           depth = k.m_priority;
       }
 
-      float const kMinDepth = -100000.0f;
-      float const kMaxDepth = 100000.0f;
-      float const d = base::Clamp(depth, kMinDepth, kMaxDepth) - kMinDepth;
+      // @todo: make sure features are prioritised the same way as in the run-time displacer,
+      // see overlay_handle.cpp::CalculateOverlayPriority()
+      ASSERT(-10000 <= depth && depth < 10000, (depth));
```

Comment:
dp::kMinDepth, dp::kMaxDepth ?

Reply:
nope its different (kMaxDepth == 25000)

Reply:
I'll add proper constants here later when refactoring this part.